### ISSUE TITLE
Keep reminders when updating events

### ISSUE THREAD

**Issue by jordanbtucker** [derekantrican/GAS-ICS-Sync#299]
When an event is updated, it does not retain any reminders previously added to it.

For example:
1. Sync an ICS with a Google Calendar.
2. Add a reminder to an event.
3. Sync the ICS again after that event was changed in the ICS.
4. Note that the event in the Google Calendar has its reminders removed.

I'd like to request a feature where reminders added to a Google Calendar event stay with that event even after being updated by a sync. This likely only makes sense when `addAlerts` is set to `false`.

**Comment by jonas0b1011001** [comment]
In Helpers.gs:

- in line [222](https://github.com/derekantrican/GAS-ICS-Sync/blob/6438b551534e191380b4f10f327ac70a90d36599/Helpers.gs#L222) replace `Calendar.Events.update` with `Calendar.Events.patch`

- comment out/remove lines [389-394](https://github.com/derekantrican/GAS-ICS-Sync/blob/6438b551534e191380b4f10f327ac70a90d36599/Helpers.gs#L389) and (if you have it) lines [424-427](https://github.com/derekantrican/GAS-ICS-Sync/blob/6438b551534e191380b4f10f327ac70a90d36599/Helpers.gs#L424).

Let me know if that's the behavior you are looking for.

**Comment by jordanbtucker** [comment]
@jonas0b1011001 Thanks. I already did something similar that worked.

I added the following line between lines [221](https://github.com/derekantrican/GAS-ICS-Sync/blob/6438b551534e191380b4f10f327ac70a90d36599/Helpers.gs#L221) and 222 and set `addAlerts` to `false`.

```js
newEvent.reminders = calendarEvents[index].reminders;
```

This is a little more narrow in scope than `patch` because I was just targeting reminders. But I could see how it could be useful to `patch` the event to maintain any other information added to an event that isn't in the ICS, not just reminders.
